map/map_loader/CMakeLists.txt

```diff
@@ -47,4 +54,5 @@ install(PROGRAMS
 ament_auto_package(INSTALL_TO_SHARE
   launch
   config
+  data
```

Comment:
Would it be possible to put the `data/` directory under `test/` and install only when `BUILD_TESTING` is `true`? I guess it can save disk space for production builds.

Reply:
Done: [7925c47](https://github.com/autowarefoundation/autoware.universe/pull/1056/commits/7925c47d0c7fcc6c4f1f7d9028e33bf76fe2053a)
For some reason I could not do `ament_auto_package(INSTALL_TO_SHARE ...)` twice so I used a list instead.

Reply:
What do you think about using the `install` command directly? cc @KeisukeShima 
https://github.com/autowarefoundation/autoware.universe/blob/64a588bd572e2cce4e8c4fd694c5c2285bfe420f/simulator/fault_injection/CMakeLists.txt#L39-L43

Reply:
Using the `install` command directly works nicely :+1: 
Now I do not need to modify the `ament_auto_package(INSTALL_TO_SHARE ...)` command.
Is that okay ? [3d6329f](https://github.com/autowarefoundation/autoware.universe/pull/1056/commits/3d6329fa424d75ba5551988d4bd40b5484767988)

Reply:
@maxime-clem By the way, are there any reasons that you used `FILES` instead of `DIRECTORY` for the `install` command?

Reply:
I am not very familiar with the `cmake` commands :sweat_smile: I switched to using `DIRECTORY`: [17e1aa6](https://github.com/autowarefoundation/autoware.universe/pull/1056/commits/17e1aa64c0a8f63bdcbddaee61362f8260773860)